### ISSUE TITLE
Add release team: openvmp

### ISSUE THREAD

**Issue by openvmp** [ros2-gbp/ros2-gbp-github-org#230]
* Name of the new release team: openvmp
* Release repositories to add:
  * [ros2_serial](https://github.com/openvmp/serial)
    * https://github.com/openvmp/serial-release
  * [ros2_serial_bus](https://github.com/openvmp/serial_bus)
    * https://github.com/openvmp/serial_bus-release

Once the above basic repositories are added, the following repositories will be prepared for releasing too:
  * [ros2_em2rs](https://github.com/openvmp/stepper_driver_em2rs)
  * [ros2_amt21](https://github.com/openvmp/encoder_amt21)
  * [ros2_r413d08](https://github.com/openvmp/switch_r413d08)
  * [ros2_modbus](https://github.com/openvmp/modbus)
  * [ros2_modbus_rtu](https://github.com/openvmp/modbus_rtu)
  * [remote_hardware_interface](https://github.com/openvmp/remote_hardware_interface)
  * [remote_actuator](https://github.com/openvmp/actuator)
  * [remote_stepper_driver](https://github.com/openvmp/stepper_driver)
  * [remote_encoder](https://github.com/openvmp/encoder)
  * [remote_brake](https://github.com/openvmp/brake)
  * [remote_switch](https://github.com/openvmp/switch)
  * [ros2_brake_switch](https://github.com/openvmp/brake_switch)

* Team members
  * [@openvmp](https://github.com/openvmp/serial/blob/main/package.xml#L7)

**Comment by nuclearsandwich** [comment]
@openvmp can you please open a PR on https://github.com/ros/rosdistro adding a source entry for the `serial` and `serial_bus` packages so that these have a chance to undergo REP-144 naming review? Once the naming review is complete, we can proceed with the release repositories. Thanks!

**Comment by nuclearsandwich** [comment]
@openvmp any update on the above request?

**Comment by openvmp** [comment]
Renaming everything to comply with REP-144 and updating downstream packages.
Should be completed this weekend or the next one.

**Comment by nuclearsandwich** [comment]
Thanks for the update! I'm going to close this issue for the time being. Please re-open it. Or comment requesting it be re-opened if you cannot when the renaming efforts are completed.

**Comment by openvmp** [comment]
@nuclearsandwich Please, re-open.
Renamed ros2_serial into remote_serial and so on.
Already created https://github.com/ros/rosdistro/pull/37165

**Comment by nuclearsandwich** [comment]
@openvmp thanks for your patience with this one. I know the review has been a yo-yo of a process and we are working on improving that across the board. One small change, I've switched to labeling requests with outstanding needs rather than closing them so that they don't get lost in the bustle.

As https://github.com/ros/rosdistro/pull/37165 is still outstanding, I have not yet opened a PR to implement these repositories.

**Comment by nuclearsandwich** [comment]
This issue is still waiting on an updated source package entry. Please re-open it, or open a new issue referencing this one, when the package has been re-submitted for naming review.
